File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/UserTokenVerifyResponse.java
package com.demo.domain.dto.http.resonse;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

@Data
public class UserTokenVerifyResponse {
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private Long userId;
    private Boolean valid;
    private String message;
}

File: demo/app/app/src/main/java/com/vectordemo/domain/dto/http/response/UserTokenVerifyResponse.kt
package com.vectordemo.domain.dto.http.response

import com.google.gson.annotations.JsonAdapter
import com.vectordemo.utils.json.WireUserIdJsonDeserializer

data class UserTokenVerifyResponse(
    @JsonAdapter(WireUserIdJsonDeserializer::class)
    var userId: String? = null,
    var valid: Boolean? = null,
    var message: String? = null
)
